taskflow.go

```diff
@@ -46,51 +42,107 @@ type RegisteredTask struct {
 func New() *Taskflow {
 	return &Taskflow{
 		Output: DefaultOutput,
-		Params: Params{},
 	}
 }
 
-// Register registers the task.
-func (f *Taskflow) Register(task Task) (RegisteredTask, error) {
+// VerboseParam returns the out-of-the-box verbose parameter which controls the output behavior.
+func (f *Taskflow) VerboseParam() BoolParam {
+	if f.verbose == nil {
+		param := f.RegisterBoolParam(false, ParamInfo{
+			Name:  "v",
+			Usage: "Verbose output: log all tasks as they are run. Also print all text from Log and Logf calls even if the task succeeds.",
+		})
+		f.verbose = &param
+	}
```

Comment:
Why the parameter is not created in the constructor? At the end it is always created:
```go
func (f *Taskflow) Run(ctx context.Context, args ...string) int {
	f.VerboseParam() // make sure that verbose parameter is registered

	flow := &flowRunner{
		output:      f.Output,
```

Reply:
II wanted to make the zero-value useful.
Refactored here: https://github.com/pellared/taskflow/pull/45/commits/547c21c4f2779989f43f2a7e49a963b23a75ff36
More info: https://golang.org/doc/effective_go#initialization